src/components/button/button.tsx

```diff
@@ -5,24 +5,29 @@ import styles from './button.module.scss';
 type Props = {
   children: ReactNode;
   className?: string;
+  btnClassName?: string;
   type?: 'primary' | 'secondary' | 'link' | 'text';
   size?: 'normal' | 'large';
   disabled?: boolean;
   onClick?: () => void;
 };
 
-const Button = ({ children, disabled, type = 'primary', size = 'normal', onClick, className }: Props) => {
+const Button = ({ children, disabled, type = 'primary', size = 'normal', onClick, className, btnClassName }: Props) => {
```

Comment:
You don't need this actually. You can make it styled using the following CSS (using the className for "wrapper" element)

```css
.someStyles {
    button {
         ...
    }
}
```